src/client/theme-default/components/VPNavBarSearch.vue

```diff
@@ -34,22 +34,20 @@ const preconnect = () => {
 
   const rIC = window.requestIdleCallback || setTimeout
   rIC(() => {
-    if (!theme.value.algolia || document.head.querySelector(`#${id}`)) return
```

Comment:
The second one is necessary to prevent adding the preconnect element multiple times in head.

Reply:
But IG it won't by triggered more than once, will it?

Reply:
what I tested was that it only triggered once